tests/test_utils.py

```diff
@@ -69,26 +70,34 @@ def test_handle_kwargs(self):
         assert (kwargs.get('uuid').startswith('20'))
         assert (kwargs.get('primary') is False)
 
+    def _env_is_up_and_healthy(env):
+        env_url = get_beanstalk_real_url(env)
+        health_page_url = f"{env_url}/health?format=json"
+        return requests.get(health_page_url).status_code == 200
+
     def test_get_s3_utils(self):
         """
         Sanity test for s3 utils for all envs
         """
         environments = [env for env in self.app_utils_obj.init_environments() if 'cgap' not in env]
```

Comment:
I would refactor this test to parametrize on the env value here, that way you do not skip the test for _all_ envs if _one_ of them is not functioning.